tools/src/main/scala/scala/scalanative/codegen/CodeGen.scala

```diff
@@ -12,6 +12,13 @@ import scalanative.nir._
 
 object CodeGen {
 
+  private final class Platform(target: String) {
+    val isWindows = target.contains("indows")
```

Comment:
We can not merge them, library and compiler can't share the code. One is Scala Native compiled and the other one is JVM compiled.